pkg/cmd/inletsoperator_app.go

```diff
@@ -30,12 +41,60 @@ func makeInstallInletsOperator() *cobra.Command {
 		namespace, _ := command.Flags().GetString("namespace")
 
 		if namespace != "default" {
-			return fmt.Errorf(`to override the namespace, edit the YAML files on GitHub`)
+			return fmt.Errorf(`to override the namespace, install inlets-operator via helm manually`)
 		}
+
+		arch := getArchitecture()
+		fmt.Printf("Node architecture: %q\n", arch)
+
+		userPath, err := config.InitUserDir()
+		if err != nil {
+			return err
+		}
+
+		clientArch, clientOS := getClientArch()
```

Comment:
Not required for any purpose I can think of?

Reply:
clientArch, and clientOS are used to download helm on the appropriate client machine. That must be the purpose? Since we're passing them in the tryDownloadHelm() function

Reply:
My mistake, I was confusing this with the server-arch.